### ISSUE TITLE
MobileNetV2

### ISSUE THREAD

**Issue by Kirill94a** [tensorflow/tpu#567]
MobileNet version 2 from tensorflow.keras does not converge on the validation set, although on the training set it gives an accuracy of about 100%, but on the validation set it is always about 50%. At the same time, MobileNet version 1 converges perfectly both on the training set and on the validation set.
